style.css

```diff
@@ -122,6 +122,11 @@ body {
   text-align: center;
   padding-top: 10px;
   padding-right: 10px;
+  background-color: white;
+}
+
+.home-button:hover {
+  background-color: rgb(4, 134, 240);
```

Comment:
Hover works well & looks good - Use in future. Try to add for images & other links in future.